addon/@ember/test-waiters/build-waiter.ts

```diff
@@ -1,4 +1,4 @@
-import { Primitive, TestWaiter, TestWaiterDebugInfo, WaiterName } from 'ember-test-waiters';
+import { Primitive, TestWaiter, TestWaiterDebugInfo, WaiterName } from '@ember/test-waiters';
```

Comment:
Can you update this to be a relative path?

Reply:
Yep.